src/validations/wearable.ts

```diff
@@ -69,7 +69,15 @@ export const wearableThumbnail: Validation = {
     const errors: string[] = []
     // check size
     const thumbnailBuffer = deployment.files.get(hash)
-    if (!thumbnailBuffer) return validationFailed(`Couldn't find thumbnail file with hash: ${hash}`)
+    if (!thumbnailBuffer) {
+      const isHashStored = (await externalCalls.isContentStoredAlready([hash])).get(hash) ?? false
+      if (!isHashStored) {
+        return validationFailed(`Couldn't find thumbnail file with hash: ${hash}`)
+      }
+      // otherwise, thumbnail was already uploaded and won't be validated again
+      log?.debug(`Thumbnail file with hash: ${hash} is not in the deployment, but it is already stored`)
```

Comment:
I think `log?.debug()` will fail if `log` is undefined

Reply:
it shouldn't, `<lhs>?.<rhs>` stops if lhs is falsy just like `<lhs> && <rhs>` does